src/test/regress/multi_schedule

```diff
@@ -30,14 +30,9 @@ test: multi_read_from_secondaries
 # uploading data to it.
 # ----------
 test: multi_create_table
-test: multi_create_table_constraints
-test: multi_master_protocol
-test: multi_load_data
-
-test: multi_behavioral_analytics_create_table
-test: multi_behavioral_analytics_basics multi_behavioral_analytics_single_shard_queries multi_insert_select_non_pushable_queries
-test: multi_insert_select multi_insert_select_window
-test: multi_shard_update_delete
+test: multi_create_table_constraints multi_master_protocol multi_load_data multi_behavioral_analytics_create_table
+
+test: multi_behavioral_analytics_basics multi_behavioral_analytics_single_shard_queries multi_insert_select_non_pushable_queries multi_insert_select multi_insert_select_window multi_shard_update_delete
```

Comment:
Could we do the parallelisation in a separate PR? This one is already quite huge, so it'd be easier to split them in order to get things merged.

Reply:
Sure